base/tests_college.py

```diff
@@ -0,0 +1,79 @@
+from django.test import TestCase
+from django.test import Client
+from base.models import *
```

Comment:
We should only need to import College for these tests, don't need all the models.